### ISSUE TITLE
can't add a new key into hash during iteration

### ISSUE THREAD

**Issue by espen** [airbrake/airbrake#904]
```
/airbrake-8.1.2/lib/airbrake/rails/action_controller_route_subscriber.rb:21→ call
/activesupport-5.2.2/lib/active_support/notifications/fanout.rb:129→ finish
```

We experienced this error after upgrading airbrake to 8.1.2 and airbrake-ruby 3.2.3. It was not caught in our tests and I am unable to reproduce it when raising some generic error. However since this error occurred in production we just had to downgrade to airbrake 8.01 and airbrake-ruby 3.1.0 where we haven't seen this issue. The error was logged in various controllers on our end and I do not see a pattern in the request (GET, PUT and POST, controller, various params etc). Sorry I don't have any more info.

**Comment by hartator** [comment]
Same here. Seems to be random affecting 30-40 requests on a couple of millions.

It seems to be linked to AirBrake last version for Ruby:

```
RuntimeError: can't add a new key into hash during iteration

/gems/airbrake-8.1.2/lib/airbrake/rails/action_controller_route_subscriber.rb:21 in call
/gems/activesupport-5.2.2/lib/active_support/notifications/fanout.rb:129 in finish
/gems/activesupport-5.2.2/lib/active_support/notifications/fanout.rb:48 in block in finish
/gems/activesupport-5.2.2/lib/active_support/notifications/fanout.rb:48 in each
/gems/activesupport-5.2.2/lib/active_support/notifications/fanout.rb:48 in finish
/gems/activesupport-5.2.2/lib/active_support/notifications/instrumenter.rb:44 in finish_with_state
/gems/activesupport-5.2.2/lib/active_support/notifications/instrumenter.rb:29 in instrument
/gems/activesupport-5.2.2/lib/active_support/notifications.rb:168 in instrument
/gems/actionpack-5.2.2/lib/action_controller/metal/instrumentation.rb:30 in process_action
/gems/actionpack-5.2.2/lib/action_controller/metal/params_wrapper.rb:256 in process_action
/gems/actionpack-5.2.2/lib/abstract_controller/base.rb:134 in process
/gems/actionview-5.2.2/lib/action_view/rendering.rb:32 in process
/gems/actionpack-5.2.2/lib/action_controller/metal.rb:191 in dispatch
```

**Comment by kyrylo** [comment]
This should be fixed in v8.1.3. 👌 
Sorry for the inconvenience.

**Comment by hartator** [comment]
Thanks @kyrylo! Just pushed to production. 👍 

```
Using airbrake-ruby 3.2.4
Fetching airbrake 8.1.3 (was 8.1.2)
Installing airbrake 8.1.3 (was 8.1.2)
```
